toolkit/hooks/src/useSlider.ts

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect, useRef } from "react";
```

Comment:
Please remove this file from Progress hook PR.

Reply:
file s removed